paasta_tools/kubernetes/bin/paasta_cleanup_remote_run_resources.py

```diff
@@ -96,9 +118,11 @@ def main():
     args = parse_args()
     logging.basicConfig(level=logging.DEBUG if args.verbose else logging.INFO)
     kube_client = KubeClient()
-    age_limit = datetime.now(tz=timezone.utc) - timedelta(seconds=args.max_age)
+    now = datetime.now(tz=timezone.utc)
+    age_limit = now - timedelta(seconds=args.max_age)
+    job_age_limit = now - timedelta(seconds=get_max_job_duration_limit())
     for namespace in get_all_managed_namespaces(kube_client):
-        clean_namespace(kube_client, namespace, age_limit, args.dry_run)
+        clean_namespace(kube_client, namespace, age_limit, job_age_limit, args.dry_run)
```

Comment:
might be nice to use kwargs for the age limit params (at least) so that it's a little more self-documenting as to what `age_limit` is being used for